HHS/OPRE-OPS
# feat: :art: adds DefaultLayout with Alert

## What changed

- adds `DefaultLayout`
- moves `Breadcrumbs` to `DefaultLayout`
- adds `Alert` to `DefaultLayout`

## Issue

#1446 

## How to test

1. Login as admin
2. Goto Agreement one
3. Goto BLI tab
4. make a change and save changes to group
5. see Alert

## Screenshots

https://github.com/HHS/OPRE-OPS/assets/4629398/93287294-8ea9-49e6-8412-f7e227c1557e

## Definition of Done Checklist
- ~~OESA: Dependency rules followed~~
- ~~Form validations updated~~
